scripts/report/get_file_name.py

```diff
@@ -18,15 +18,23 @@
 import os
 
 def get_name(run_path, output_file, include_only=False):
```

Comment:
Tried to make the function more readable. Now it returns an exception if the file requested was not found